python-client/giskard/scanner/result.py

```diff
@@ -123,3 +126,22 @@ def _track_suite(self, suite, name):
             "scan:generate_test_suite",
             {"suite_name": anonymize(name), "tests_cnt": len(suite.tests), **tests_cnt},
         )
+
+    def to_mlflow(self, client=None, run_id=None, summary=True, model_artifact_path=""):
```

Comment:
I'd rename `client` to `mlflow_client` and add types to the arguments to avoid confution with GiskardClient